src/Resources/config/features/cart.yml

```diff
@@ -4,16 +4,26 @@ services:
         autoconfigure: false
         public: false
 
-    sylius.google_tag_manager_enhanced_ecommerce.cart.block_event_listener.sylius.shop.product.show.before_add_to_cart:
-        class: Sylius\Bundle\UiBundle\Block\BlockEventListener
+    sylius.google_tag_manager.enhanced_ecommerce_tracking.cart.listener.add_to_cart:
```

Comment:
Can you mention these changes also in the upgrade guide? If we are changing public API's / services names, that might impact users their overrides.